app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -38,6 +40,7 @@ def edit
 
     # GET /concern/parent/:parent_id/file_sets/:id
     def show
+      workflow_check(parent_id: params[:parent_id])
```

Comment:
do we need to check workflow permissions on `show`?

Reply:
In my reading of #3775 (see block quote), I believe it is required.

> When depositing a work into a mediated deposit workflow, the work view page is hidden to the public, but you can download the file via direct link to the fileset page.